package.json

```diff
@@ -36,8 +35,10 @@
     "cookie-parser": "^1.4.3",
     "cross-env": "^3.2.4",
     "css-loader": "^0.27.3",
+    "event-source-polyfill": "0.0.11",
```

Comment:
It's not clear what changes necessitated adding this dependency.

Reply:
It's always technically been a dependency to support webpack on ie.  For all intents and purposes it's been pointless but the config i copied it from uses it so I left it in